components/TaxilaEventsList.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div style="border-radius: 10px;" class="px-8 py-6 bg-white text-left max-w-2xl mx-auto mb-12 sm:mb-20">
+    <div id="dev-tess-widget-materials-title" style="font-size: 24px;" class="mb-2">Potentially Interesting Events (from <a href='https://taxila.nl'>taxila.nl</a>)</div>
+    <link rel="stylesheet" property="stylesheet" href="https://elixirtess.github.io/TeSS_widgets/css/tess-widget.css"/>
+    <div id="dev-tess-widget-materials-list" class="tess-widget tess-widget-list"></div>
+    <div class="flex justify-end">
+      <a href="https://taxila.nl" class="btn btn-outline btn-sm hover:bg-primary">
+        Show more
+      </a>
+    </div>
+    <script>
+      class MyCustomRenderer {
+        constructor(widget, element, options) {
+          this.widget = widget;
+          this.element = element;
+          this.options = options;
+        }
+
+        initialize() {
+          const title = document.createElement('h1');
+          this.element.appendChild(title);
+          this.list = document.createElement('ul');
+          this.element.appendChild(this.list);
+        }
+
+        render(errors, data, response) {
+          this.list.innerHTML = '';
+          data.data.forEach((event) => {
+            const eventElement = document.createElement('li');
+            const title_div = document.createElement('a');
+            title_div.textContent = event.attributes['title'];
+            title_div.href = event.attributes['url'];
+            eventElement.appendChild(title_div);
+            const start_div = document.createElement('div');
+            start_div.textContent = parseTime(event.attributes['start']);
+            start_div.classList.add('text-sm');
+            start_div.classList.add('text-gray-400');
+            eventElement.appendChild(start_div);
+            const venue_div = document.createElement('div');
+            venue_div.textContent = event.attributes['online'] ? 'Online' : event.attributes['venue'];
+            venue_div.classList.add('text-sm');
+            venue_div.classList.add('text-gray-400');
+            eventElement.appendChild(venue_div);
+            this.list.appendChild(eventElement);
+          });
+        }
+      }
+
+      function parseTime(datestring) {
+        const options = { month: 'long', day: 'numeric', year: 'numeric' };
+        const formattedDate = new Date(datestring).toLocaleDateString('en-US', options);
+        return formattedDate;
+      }
+
+      function initTeSSWidgets() {
+        TessWidget.Events(document.getElementById('dev-tess-widget-materials-list'),
+          MyCustomRenderer,
+          {
+            opts: {
+              enableSearch: true,
+              truncateLength: 2
+            },
+            params: {
+              q: "science OR technology OR python OR programming OR (research software)",
```

Comment:
research software OR open science OR programming OR software

Reply:
if research software is included then software is inherently also included. otherwise done